packages/rabbitmq/src/utils.ts

```diff
@@ -0,0 +1,128 @@
+import { CSL, Cardano, cslToCore } from '@cardano-sdk/core';
+import { OutsideOfValidityInterval } from '@cardano-ogmios/schema';
+import { fromSerializableObject, toSerializableObject } from '@cardano-sdk/util';
+
+export interface DeserializedError {
```

Comment:
Wouldn't Cardano.TxSubmissionError work as a type instead of this?

Reply:
Actually, once applied [this change](https://github.com/input-output-hk/cardano-js-sdk/pull/290#discussion_r899802463), yes, it can be discarded.

Reply:
removed